src/services/UserService.js

```diff
@@ -176,6 +179,60 @@ const deleteReservations = async user => {
   }
 };
 
+/**
+ * Creates the handle for the user
+ *
+ * @param {*} user
+ * @param {*} claimedPosition
+ */
+const getUserHandle = async (user, claimedPosition, maxQuePosition) => {
```

Comment:
I would rename this to "create" because thats more accurate.

Reply:
Yeah, you are right. Done 😁